### ISSUE TITLE
Extended Menu Style - Level 4 menu items not shown properly

### ISSUE THREAD

**Issue by racingkars** [gantry/gantry5#2457]
Hi Guys,  According to Ivo from InspireTheme this problem comes up again. It was solved in version 5.4.6.

Here the old info:  https://github.com/gantry/gantry5/issues/1778

I am not a specialist so to me it's more easy to refer to the old problem for the information on this.

Best regards,

Kars

**Comment by mahagr** [comment]
@JoomFX Do you have more information about this one?

**Comment by JoomFX** [comment]
@mahagr Hi Matias,

What information do you need?

Please watch the video in the [old issue](https://github.com/gantry/gantry5/issues/1778), it shows everything in details.

Djamil fixed the issue but it seems that some merge has overwritten the fix. I'm really not sure.

**Comment by mahagr** [comment]
@JoomFX Nope, the code is still there:

https://github.com/gantry/gantry5/blob/8ec9dc61b556c34158dec4636211e94bc6fb3f1e/assets/common/application/menu/index.js#L248

**Comment by JoomFX** [comment]
Yep, the code is still there but the issue I documented [in the video](https://youtu.be/yyjABaqz-Lc) is still reproducible.
